nana/modules/dictionary.py

```diff
@@ -15,7 +14,7 @@
 __MODULE__ = "Dictionary"
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command("dic", Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("dic", Command))
```

Comment:
Codacy found an issue: [dictionary is too complex (24) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)